External/SPMmodified/xASL/xASL_im_DistanceTransform.m

```diff
@@ -9,7 +9,7 @@
 %
 % OUTPUT:
 %  dist     - distance map, distance in voxels to the closest point on the mask
-%  x,y,z    - X, Y, and Z coordinate of the closest voxel on the mask
+%  x,y,z    - X, Y, and Z coordinate of the closest voxel on the mask. Note that these are relative coordinates, and Y,X,Z
```

Comment:
What do you mean by Y,X,Z? Shouldn't it be X Y Z? The fact that some software like SPM or Matlab turns X & Y around doesn't mean that we should do that too :)

Reply:
OK but then it should be X,Y,Z now right?

Reply:
This comment is of course shows the state from when you made the comment. It's not updating with the later commits.